### ISSUE TITLE
Option flags for crypto transfer, that are contextually correct and that follow unix/linux conventions 

### ISSUE THREAD

**Issue by calvinchengx** [hashgraph/hedera-cli#48]
**Problem**

Currently, the option flags are inconsistent, does not necessarily follow unix/linux conventions and may not be named based on the specific context of a command invocation.

**Solution**

Revise all option flags and ensure that the flags are contextually appropriate for each function, and follows unix/linux conventions to reduce cognitive load for developers.

These are some proposed examples of easy-to-remember and contextually accurate commands:

```
transfer -s=0.0.1001 -r=0.0.1002 -hb=1000
transfer --senders=0.0.1001 --recipients=0.0.1002 --hbars=1000
transfer -s=0.0.1001 -r=0.0.1002 -tb=1000`
transfer --senders=0.0.1001 --recipients=0.0.1002 --tinybars=1000 -y

# or

transfer --senders=0.0.1001,0.0.1002 --recipients=0.0.1003 --hbars=[-50,-50,100]

# or

transfer -s=0.0.1001 -r=0.0.1002,0.0.1003 --hbars=[-100,70,30]
```

This is also much simpler from an end user perspective than having `transfer single` versus `transfer multiple`.

**Comment by calvinchengx** [comment]
@enerestar 

per our discussion, the crypto transfer commands should look like:-

```
# single transfer
# usage of the "-y" flag indicates that the user wants to skip the confirmation step 
# in the preview and execute the crypto transfer immediately; the preview will still 
# be shown by the transaction is executed without pausing for confirmation

transfer --senders 0.0.1001 --recipients 0.0.1002 --hbars 10_000 --yes
transfer -s 0.0.1001 -r 0.0.1002 -hb 10_000 -y

# one sender, multiple recipients

transfer -s 0.0.1001 -r 0.0.1002,0.0.1003,0.0.1004 -hb 10_000,10_000,10_000

# multiple senders, one recipient

transfer -s 0.0.1001,0.0.1002 -r 0.0.1003 -hb -10_000,-20_000,30_000

# multiple senders, multiple recipients

transfer -s 0.0.1001,0.0.1002 -r 0.0.1003,0.0.1004 -hb -10_000,-20_000,30_000,10_000
```

**Comment by calvinchengx** [comment]
For the single sender to single recipient scenario, both of these commands should be valid:-

```
# valid
transfer -s 0.0.1001 -r 0.0.1002 -hb 10_000 -y

# valid
transfer -s 0.0.1001 -r 0.0.1002 -hb [-10_000,10_000] -y
```

Right now, `transfer -s 0.0.1001 -r 0.0.1002 -hb 10_000 -y` is invalid because they mean exactly the same thing.

**Comment by calvinchengx** [comment]
Resolved with PR #103
